docs/jwst/pathloss/description.rst

```diff
@@ -6,23 +6,30 @@ Overview
 --------
 
 The pathloss correction step calculates the correction to apply to spectra
-when the 1-d extraction is performed.  This correction accounts for losses
+when the 1-d extraction is performed.  The motivation behind the correction
+is different for NIRSPEC and NIRISS, while that for MIRI has not been
+implemented in Build 7.3.  For NIRSPEC, this correction accounts for losses
```

Comment:
You mean Build 7.2 here (7.3 isn't until next Spring)

Reply:
OK, I'll just replace with 'yet'.